src/main/java/billgates/view/gui/BillPanel.java

```diff
@@ -55,4 +51,16 @@ private void billClick() {
     public JTable getBillTable() {
         return this.billTable;
     }
-}
+
+    // Change color of table header and border in bill panel
+    public void changeColor(Color c){
+        billTable.getTableHeader().setBackground(c);
```

Comment:
Convention: `this.` should be added in front of the instance variable `billTable`.